dockers/docker-database/Dockerfile.j2

```diff
@@ -48,5 +48,6 @@ COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
 COPY ["files/sysctl-net.conf", "/etc/sysctl.d/"]
 COPY ["files/update_chassisdb_config", "/usr/local/bin/"]
 COPY ["flush_unused_database", "/usr/local/bin/"]
+COPY ["multi_database_config.json.j2", "/usr/share/sonic/templates/"]
```

Comment:
@Pan-XT Could you upload a write-up for this multi-DB including the current issues and usage.

Reply:
https://github.com/dzhangalibaba/SONiC/blob/master/doc/database/multi_database_instances.md

Reply:
#Closed

Reply:
@Pan-XT Could you update the HLD with some performance numbers with the multi DB testing?

Reply:
@Pan-XT Also provide a table with number of redis instances used during testing and what is the ideal number of instances used.